org.grisbi.Grisbi.yml

```diff
@@ -0,0 +1,62 @@
+app-id: org.grisbi.Grisbi
+runtime: org.gnome.Platform
+runtime-version: "43"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.gnome.Sdk.Locale
+command: grisbi
+
+rename-desktop-file: grisbi.desktop
+
+finish-args:
+  - --socket=wayland
+  - --socket=x11
+  - --filesystem=host
```

Comment:
this is a very broad filesystem permission for a "Personal finance manager"

Reply:
Right, I'll switch to "home". I thought users may want to import the account files they get from their bank's online website. Home is sufficient for that, do you think I should go with `xdg-download`?

Reply:
more restrictive is better.. always.